### PR TITLE
update labels, save groups in session, change default consignment sort - do not use

### PR DESCRIPTION
**Pull request by colinbowen**
## Changes in this PR
- Last modified > Record date (tables + filters panel)
- Closure expiry > Record opening date (tables + filters panel)
- 'Records found XXX'. >>> Browse records XXX'. (On browse pages)
- SU text changed to "Search by file name, transferring body, series or consignment reference."
- AAU text changed to "Search by file name, series or consignment reference."
- Sort box text updates & consignment default sort changed to record date

## JIRA ticket
https://national-archives.atlassian.net/browse/AYR-718

### After
Screenshot 2024-02-16 at 10 15 52

- [ ] Requires env variable(s) to be updated

### CODE REVIEW

**Comment by pgandhizaizi** [comment]
please get the latest changes from main ( changes for top-search by Candace ) and then verify that if 'post' method needs to be replaced with 'get' on any <form> object

File: app/tests/test_sign_in.py [hunk 1]
```diff
@@ -14,3 +16,51 @@ def test_sign_in(mock_keycloak, client):
         redirect_uri="http://localhost/callback",
         scope="group_mapper_client_scope",
     )
+
+
+@patch("app.main.routes.get_keycloak_instance_from_flask_config")
+def test_callback_route_superuser(mock_keycloak, client):
```
Comment (anthonyhashemi) [inline app/tests/test_sign_in.py:22]: cheers for adding these tests for the callback route

**Review by anthonyhashemi — APPROVED** [review]
LGTM

**Comment by colinbowen** [comment]
Please see: https://github.com/nationalarchives/da-ayr-beta-webapp/pull/229. Which has conflicts resolved.

**Pull request closed without merging**